ArduCopter/compassmot.cpp

```diff
@@ -261,5 +261,5 @@ MAV_RESULT Copter::mavlink_compassmot(const GCS_MAVLINK &gcs_chan)
     ap.compass_mot = false;
 
     return MAV_RESULT_ACCEPTED;
-#endif  // FRAME_CONFIG != HELI_FRAME
+#endif  // APM_BUILD_TYPE(APM_BUILD_Heli)
```

Comment:
```suggestion
#endif  // !APM_BUILD_TYPE(APM_BUILD_Heli)
```

Reply:
The if that the comment is for doesn't have a not

Reply:
Its an if APM_BUILD_TYPE(APM_BUILD_Heli)

Reply:
this is in the else for that if.